test/e2e/tools/tools_test.go

```diff
@@ -17,6 +17,8 @@
 package tools
 
 import (
```

Comment:
`controller-manager` is the ugly default name for operator-sdk projects that yours truly forgot to change initially and now we're kinda stuck with it. Let's call it "operator" or "manager" if we feel fancy. Please **DO NOT** resubmit just to fix this silly nit I'm raising

Reply:
no problem, fixed, thanks for raising